### DISCUSSION TITLE
短影音紅利時代來臨，人人都有機會利用短影音迅速收獲曝光

### DISCUSSION THREAD

**Discussion by Valentinaa0214** [1101-GS3073-C/Python-Community-GS3073-C#681] [General]
近期無論是在IG、Youtube還是臉書等社群媒體都出現了類似tiktok的短影音功能，簡易的操作(往下滑就可以看下一部)與節奏快速的內容讓我常常忍不住一部接著一部看，不知不覺就看了三、四個小時。
雖然這種形式的影音能過獲得大量的觀看數、流量，但比較難讓我在看完後留下比較深刻的印象。
大家喜歡這種新形式的短影音嗎？對此的看法如何？
https://www.kolradar.com/blog/trend-sharing/kol-marketing-trend/short-video-content/
https://www.thenewslens.com/article/152553

**Comment by Aerofe** [comment]
短影音真的會讓人一直想要看下去，很常滑一滑就感覺時間就過了😂

**Comment by ying0607** [comment]
短影音確實會讓人一直想接著看，但畢竟是‘’短‘’影音，以舞台劇舉例的話，比較像在看表演的highlight，沒有看完整劇情的感覺

**Comment by 0212Jerry** [comment]
短影音之所以如此氾濫我覺得有一個特點，儘管這部影片品質再差，也只有不到一分鐘的時間，若遇到很好的影片也會趨勢人繼續往下看。此外，這也方便原po 散佈自己的內容。
